modules/alloy/src/main/ts/ephox/alloy/behaviour/blocking/BlockingState.ts

```diff
@@ -0,0 +1,42 @@
+import { Cell, Optional } from '@ephox/katamari';
+import { AlloyComponent } from '../../api/component/ComponentApi';
+import { BehaviourState, nuState } from '../common/BehaviourState';
+
+export interface BlockingState extends BehaviourState {
```

Comment:
Readonly and move this to a types file please

Reply:
as above

Reply:
Again, this has not been made readonly.